applications/serial_lte_modem/doc/nRF91_as_Zephyr_modem.rst

```diff
@@ -0,0 +1,114 @@
+.. _slm_as_zephyr_modem:
```

Comment:
I think that it would be a good idea to link this page from the CMUX AT-command. To kind of tie it that this is the scenario where you would usually use CMUX.

Reply:
It is a good idea. Although everything is made to be usable independently, the features shouldn't bee too hidden from each other either.
(And actually another scenario I think is possible is to use CMUX and PPP but through something else than Zephyr's cellular modem driver. CMUX and PPP is the kind of typical functionality present in modems.)